package.json

```diff
@@ -128,6 +128,7 @@
     "metro-runtime": "0.73.3",
     "metro-source-map": "0.73.3",
     "mkdirp": "^0.5.1",
+    "mock-fs": "^5.1.4",
```

Comment:
QQ: Is this should be under devDependencies?

Reply:
it was but we have trouble in CI and in release branch due to our (improper) monorepo setup. :(

Reply:
ah actually @jacdebug you are correct, it should be under DevDeps, let me rectify